thorlcr/slave/slavmain.cpp

```diff
@@ -757,13 +759,8 @@ class CKJService : public CSimpleInterfaceOf<IKJService>, implements IThreaded,
                 unsigned rowCount = getRowCount();
                 unsigned rowNum = 0;
                 unsigned rowStart = 0;
-                const CRuntimeStatisticCollection & stats = kmc->queryService().contextLogger.queryStats();
-                unsigned __int64 startSeeks = stats.getStatisticValue(StNumIndexSeeks);
-                unsigned __int64 startScans = stats.getStatisticValue(StNumIndexScans);
-                unsigned __int64 startWildSeeks = stats.getStatisticValue(StNumIndexWildSeeks);
-                unsigned __int64 startNodeDiskFetches = stats.getStatisticValue(StNumNodeDiskFetches);
-                unsigned __int64 startLeafDiskFetches = stats.getStatisticValue(StNumLeafDiskFetches);
-                unsigned __int64 startBlobDiskFetches = stats.getStatisticValue(StNumBlobDiskFetches);
+                CStatsContextLogger  & contextLogger = kmc->queryContextLogger();
```

Comment:
Bye the way, this section didn't work before. There was multiple threads were using the same contextLogger.